db/migrations/3_create_column_groups_attempts_iMinusScore.sql

```diff
@@ -0,0 +1,23 @@
+-- +migrate Up
+ALTER TABLE `groups_attempts` ADD COLUMN `iMinusScore` float;
+UPDATE `groups_attempts` SET iMinusScore = -iScore;
```

Comment:
1. `iMinusScore` column is not needed if we move to MySQL 8.0 (since there are sorted indices and the goal of these migrations is to `ORDER BY idGroup, idItem, iScore DESC, sBestAnswerDate` without the filesort), so it is just for now (at least, I hope so).
2. I modified triggers to set iMinusScore everywhere automatically.
3. I would prefer renaming all the columns at once and use the old naming style for all the columns (even for new once) until we move to some new naming style. This way we will have consistent naming everytime.

Reply:
2. I thought you didn't like triggers   😉
3. ok

Reply:
I still don't like triggers